base/broadcast.jl

```diff
@@ -283,6 +288,15 @@ function broadcast_c(f, ::Type{Tuple}, As...)
     n = length(shape[1])
     return ntuple(k->f((_broadcast_getindex(A, k) for A in As)...), n)
 end
+function broadcast_c(f, ::Type{Nullable}, a...)
+    nullity = any(isnull, a)
```

Comment:
Have a look at `hasnulls` here: https://github.com/JuliaStats/NullableArrays.jl/pull/166/files#diff-2d8d5b5a5cc004c730be016fd93348d8R68

In general, I think you should be able to steal lots of ideas from that PR, since it's very similar in many respects. The [varargs implementation of `broadcast_lift`](https://github.com/JuliaStats/NullableArrays.jl/pull/166/files#diff-2d8d5b5a5cc004c730be016fd93348d8R82) is about 4x slower than the hand-written version for three-arguments, and it doesn't allocate. I'm still looking for ways to reduce the difference (maybe with a generated function).